models/fieldtype_blackbox_test.go

```diff
@@ -1,19 +1,19 @@
-// +build unit
-
-package models
+package models_test
 
 import (
 	"testing"
+
+	. "github.com/almighty/almighty-core/models"
 	"github.com/almighty/almighty-core/resource"
 )
 
 var (
-	stString   = SimpleType{KindString}
-	stInt      = SimpleType{KindInteger}
-	stFloat    = SimpleType{KindFloat}
-	stDuration = SimpleType{KindDuration}
-	stURL      = SimpleType{KindURL}
-	stList     = SimpleType{KindList}
+	stString   = SimpleType{Kind: KindString}
```

Comment:
To be honest: I'm not sure. Most of the time to be explicit is a good thing in Go. That's probably why I've written it out.